Rakefile

```diff
@@ -98,6 +98,11 @@ task :generate_carthage_project do
   generate_carthage_project()
 end
 
+desc "Generate Sourcery files"
+task :sourcery do
+  sh "mint run krzysztofzablocki/Sourcery sourcery"
```

Comment:
should pin to specific version of Sourcery?

Reply:
Yep @toshi0383. I'm waiting for this to be merged to pin to the proper version https://github.com/krzysztofzablocki/Sourcery